portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,10 +54,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      num_cmnts++;
-      String Cmt = request.getParameter("cmnt");
-      comments.add("#Comment"+num_cmnts+": "+Cmt);
-      response.sendRedirect("/index.html");
+    num_cmnts++;
+    String Cmt = request.getParameter("cmnt");
+    comments.add("#Comment"+num_cmnts+": "+Cmt);
+
+    Entity taskEntity = new Entity("Comment");
+    taskEntity.setProperty("Number", num_cmnts);
+    taskEntity.setProperty("text", comments.get(num_cmnts-1));
```

Comment:
Nit: Use the same case convention for all properties

Reply:
changed number to lower case